src/Client/StripeClient.php

```diff
@@ -193,7 +193,7 @@
  * @method ResourceIterator getBalanceTransactionsIterator()
  * @method ResourceIterator getEventsIterator()
  */
-class StripeClient extends Client
+final class StripeClient
```

Comment:
Don't make a class `final` unless you have an implemented interface for it (makes mocking and composition impossible)